conf/cut_release.sh

```diff
@@ -125,6 +125,9 @@ build_assets() {
     printf "${blue}Building assets...${end}"
     yarn build:npm || return 1
     printf "${green}Built assets!${end}"
+    printf "${blue}Building locales...${end}"
+    yarn build:i18n || return 1
```

Comment:
When you run i18n build after npm build, the properties file will get updated. If the file gets updated which means someone missed i18n build when pushing.

Reply:
This change should probably be in a separate PR.